core-bundle/src/Resources/contao/themes/flexible/basic.css

```diff
@@ -518,10 +520,15 @@ a.tl_submit {
 	width:100%;
 }
 .m12 {
-	margin:17px 15px;
+	margin:0 15px;
+	padding:17px 0;
 }
 .cbx {
-	min-height:46px;
+	min-height:47px;
+}
+.cbx.m12 {
```

Comment:
Since the class attribute will be `class="w50 m12 widget cbx"`, is it more performant to use `.m12.cbx` instead of `.cbx.m12` or does it not matter at all? @ausi /cc

Reply:
I didn't even know that this might matter in any way? 😁

Reply:
The order of CSS class names does not matter for performance.